.travis/run_task.py

```diff
@@ -52,6 +52,8 @@ def main():
         make('loris-publish')
     elif task == 'miro_preprocessor-test':
         make('miro_preprocessor-publish')
+    elif task == 'lambdas-test':
+        make('lambdas-publish')
```

Comment:
Do we want to publish on test or is the task misnamed?

Reply:
This only runs if we've pushed to master – let me see if I can make the code a little clearer.